backstop/tests/toast-notification.html

```diff
@@ -30,13 +30,166 @@
     </style>
     <script src="../assets/theme.js"></script>
   </head>
-  <body class="iui-body">
+  <body
+    class="iui-body"
+    style='height: 100vh;'
+  >
     <theme-button></theme-button>
     <h1>Toast notification</h1>
     <hr />
+    <div class="iui-input-container iui-inline">
+      <div class="iui-label iui-required">Toast notifications placement</div>
+      <label
+        class="iui-radio"
+        id='top-start-option'
+      >
+        <input
+          type="radio"
+          name="radio-demo-6"
+          value="top-start"
+        />
+        <span class="iui-radio-dot">
+          <svg
+            viewBox="0 0 16 16"
+            aria-hidden="true"
+            focusable="false"
+          >
+            <circle
+              cx="8"
+              cy="8"
+              r="6"
+            />
+          </svg>
+        </span>
+        <span class="iui-label">Top Start</span>
```

Comment:
Can we relabel these so it says Top Left instead of Top Start?  Same goes with Top End to Top Right.

Reply:
On the contrary, I would just change the text to lower-kebab-case, i.e. `top-start`, `top-end`, etc. It better matches the class names and the prop values in react too.